clause_6_13_ModelTextures.adoc

```diff
@@ -21,6 +21,19 @@ In OpenFlight, several types of textures can be applied in various combinations.
 
 Base textures footnote:[The CDB Standard uses the term “base texture” the same way as OpenFlight and Creator do.] are set of mutually exclusive model textures that provide texture color/intensity modulation for the model. While a model can have many base textures, only one base texture can be referenced and applied to model geometry at a time.
 
+The base texture, by default is meant to represent a diffuse map. In contrast with an Albedo texture, the diffuse texture can contain the effects of how light is scattered across the surface of the object.
+
+To clarify the intended interpretation and rendering result of the base texture when using Physically Based Rendering (PBR) concepts like Roughness and Metalness (RM), the following table provides guidance on the various use-cases. Note: the Ambient occlusion component (O) is left optional.
+
+|===
+|(base) |(O)RM| Albedo| Legacy System | PBR System
```

Comment:
Because the table cells below use "Diffuse:" rather than base, suggesting that we either change the header above to  `Diffuse (base)`, *OR* change the `Diffuse:` below representing _which texture_ (_not_ the Diffuse after the `:` which is the interpretation) to _Base_

Reply:
Jerome do you mind drafting an alternate table; I admit to having trouble visualizing it?

Reply:
https://docs.google.com/document/d/1w0MBTVxF-GADDIsNXoG6XK-OjKXWX3REy3WLe2Eekp0/edit

Reply:
@DavidFLOR sure I can do that. Would it be easier if you merge the PRs and I submit another PR with the improvements?

Reply:
Thanks Jerome, I think that is a great suggestion.  I feel kind of stuck with a PR already passed an eVote and continuing to revise it without another vote.  With the updated table, if I don't receive any other input by Friday, I will merge the two PRs, and then it would be great if you have further suggestions to put them in a PR to brief at the next SWG meeting.